.github/workflows/template_techdocs_monorepo.yml

```diff
@@ -0,0 +1,73 @@
+name: TechDocs
+
+on:
+  workflow_call:
+    secrets:
+      azure-account-name:
+        required: false
+      azure-account-key:
+        required: false
+jobs:
+  techdocs:
+    name: TechDocs
+    runs-on: ubuntu-24.04
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - uses: actions/setup-node@v4.2.0
+      - uses: actions/setup-python@v5
+        with:
+          python-version: "3.12"
+
+      - name: Install techdocs-cli
+        run: sudo npm install -g @techdocs/cli
+
+      - name: Install mkdocs and mkdocs plugins
+        run: |
+          python -m pip install mkdocs-techdocs-core==1.* mkdocs-awesome-nav==3.0.0
```

Comment:
We will also `mkdocs-awesome-pages-plugin==2.*` otherwise we would need to migrate all repos for `mkdocs-awesome-nav` first

Reply:
Thanks, I added the dependency.